src/modules/helpers/unique.ts

```diff
@@ -81,7 +81,7 @@ Try adjusting maxTime or maxRetries parameters for faker.helpers.unique().`
  * @param options.compare The function used to determine whether a value was already returned. Defaults to check the existence of the key.
  * @param options.store The store of unique entries. Defaults to `GLOBAL_UNIQUE_STORE`.
  */
-export function exec<Method extends (...parameters) => RecordKey>(
+export function exec<Method extends (...parameters: any[]) => RecordKey>(
```

Comment:
This can be expressed in a more generic way, to avoid explicit any (same in index.ts):

```suggestion
export function exec<Method extends <T>(...parameters: T[]) => RecordKey>(
```

@ST-DDT, can you confirm this? I'm "only" 90% sure that it works as expected.

Reply:
NVM, I had massive brain lag or something... this doesn't work as we require the parameters generically as well. MB.